pkg/broker/broker.go

```diff
@@ -53,9 +54,46 @@ func NewAnsibleBroker(
 		engine:        &engine,
 	}
 
+	// If no openshift target is provided, assume we are running in an openshift
+	// cluster and try to log in using mounted cert and token
+	inCluster := clusterConfig.Target == ""
+	if inCluster {
+		err := broker.Login()
+		if err != nil {
+			return broker, err
+		}
+	}
+
 	return broker, nil
 }
 
+func (a AnsibleBroker) Login() error {
```

Comment:
there might be, though this function only needs to be called once while the refreshLoginToken needs to be called every run. Maybe I could pull out the shared functionality into its own method and have both call it.